src/main.rs

```diff
@@ -1,11 +1,12 @@
 use clap::{command, Arg, ArgMatches, Command};
+use std::process::ExitCode;
 
 pub mod api;
 pub mod commands;
 
 mod ui;
 
-fn main() {
+fn main() -> ExitCode {
   let matches = command!("clink")
```

Comment:
Definitely check out the clap 3 derive feature, it makes everything so much nicer, it's especially better than manually digging through ArgMatches

Reply:
I've been using this on another project... I've been meaning to look at that!